src/entities/elastic-storage/elastic-storage-processor.js

```diff
@@ -27,66 +76,165 @@ class ElasticStorageProcessor implements StorageProcessor {
 
   networkStartTime: number
 
+  elasticConfig: ElasticConfigType
+
   constructor(
     logger: Logger,
-    elasticNode: string,
+    elasticConfig: ElasticConfigType,
     networkConfig: NetworkConfig,
   ) {
     this.logger = logger
-    this.client = new Client({ node: elasticNode })
+    this.elasticConfig = elasticConfig
+    this.client = new Client({ node: elasticConfig.node })
     this.networkStartTime = networkConfig.startTime()
   }
 
+  indexFor(name: string) {
+    // TOFO: memoize
+    return `${this.elasticConfig.indexPrefix}.${name}`
+  }
+
+  async rollbackTo(height: number) {
+    await sleep(10000)
+    const latestStableChunk = await this.getLatestStableChunk()
+    return this.deleteChunksAfter(Math.min(latestStableChunk, height))
+  }
```

Comment:
Here we are comparing block height with a chunk. There should be an additional function that queries the block chunk by the height argument, and then ew will compare two chunk numbers.